website/benutzerhandbuch.md

```diff
@@ -0,0 +1,336 @@
+---
+title: Benutzerhandbuch
+date: '2018-11-28T15:14:39+10:00'
+layout: default_5
+intro_image: null
+intro_image_absolute: false
+intro_image_hide_on_mobile: true
+---
+
+**Um eigene Toppings zu publizieren, muss man sie lokal erstellen und testen. Zukünftig sollen auch Tools einige Arbeitsschritte (wie zum Beispiel Layertree Export) übernehmen. Bis dahin sollen die folgenden Punkte ausgeführt werden, wie am Beispiel des Models Nutzungsplan Luzern**
+
+## Realisierung Toppings für Nutzungsplanung_v310
+Folgende Toppingfiles werden erstellt bzw. angepasst
+
+- ilidata.xml (wird manuell angepasst)
+- Metakonfigurationsfile (wird manuell erstellt)
+- Layerkonfigurationen QML-Files (werden mit QGIS generiert)
+- Layerreihenfolge und Legende YAML-File (wird manuell erstellt)
+- Zur Projketerstellung benötigtes ili2db Metaattributfile TOML-File (wird manuell erstellt)
+
+
+### 1. Vorbreitung
+Es ist empfehlenswert, die Dateien in einem Verzeichnis abzulegen und folgende Unterverzeichnisse zu benennen:
+
+```
+.
+├── ilidata.xml
+├── layertree
+├── metaconfig
+├── qml
+├── referencedata
+├── sql
+└── toml
+```
+
+#### Lokale Projektgenerierung
+Dieser Ordner kann dann lokal im QGIS Model Baker getestet werden. `ilidata.xml` und `ilimodels.xml` werden darin gesucht und geparst.
+
+![localrepo](../images/localrepo.png)
```

Comment:
```suggestion
![localrepo](_services/images/localrepo.png)
```

Reply:
Or better - since the localrepo.png is used on very different locations (as well in konzept) - to move the file into the `images` folder.